### ISSUE TITLE
关于ce损失的问题

### ISSUE THREAD

**Issue by qishibo** [vdogmcgee/SimCSE-Chinese-Pytorch#2]
https://github.com/vdogmcgee/SimCSE-Chinese-Pytorch/blob/main/simcse_unsup.py#L142-L145

![image](https://user-images.githubusercontent.com/5136418/145787542-55ee890b-652b-437e-b278-6ce5f1e8d694.png)

这里求对比损失的时候又在loss外面包了一层`torch.mean(loss)`，ce默认已经是`mean`均值形式了，为什么返回的时候又执行mean操作呢

**Comment by vdogmcgee** [comment]
感谢指正 , 这一步是有点多余了 , 没必要再算一次
